frontend/src/components/processMeasurements/ProcessMeasurementsDetailContent.js

```diff
@@ -46,6 +52,15 @@ const ProcessMeasurementsDetailContent = ({
       listPropertyValues({ object_id__in: processMeasurement.id, content_type__model: "processmeasurement" });
     }
 
+    const processIsLoaded = processMeasurement?.process in processesByID;
+    const process = processesByID[processMeasurement?.process] || {};
+    if (!processIsLoaded) {
```

Comment:
This block of code which is retrieving data and updating the redux state needs to be wrapped in useEffect(). We are not supposed to update state during render.